src/content/FAQ_en.md

```diff
@@ -1,323 +1,440 @@
-</details>
+Editing Hospital Care Data – [Table Pacienti (Patients)](https://docs.google.com/spreadsheets/d/1N1qLMoWyi3WFGhIpPFzKsFmVE0IwNP3elb_c18t2DwY/edit#gid=918589010):
+
+- We receive daily reports and monitor the announcements of all COVID-19 hospitals  (UKC Ljubljana, UKC Maribor, UK Golnik, SB Celje) – at around 8 am.
+
+- We monitor the number of hospitalizations: all wards, in ICUs, and on ventilators.
 
+- We also record transitions (acceptance/dismissal) between individual stages of the disease (when detectable) from the obtained data.
+
+- Where the transition (admission/dismissal) information is incomplete, the values are determined by inference (using a formula).
+
+- All sources and conclusions are recorded as a commentary in individual cells (checkable).
+
+- The data is compared with the summary data on hospitalized patients in ICU published by the Government of the Republic of Slovenia.
+  
+  </details>
 
 <details>
-  <summary>Na drugih portalih so številke obolelih drugačne kot pri vas. Zakaj?</summary>
+  <summary id=data-publish-time>When do you publish the data? Why are the dates on the visualizations different?
+</summary>
 
-Sledilnik uporablja zgolj potrjene, uradne podatke, ki jih dnevno sporočajo NIJZ in vse slovenske bolnišnice, ki zdravijo bolezen COVID-19. Naši podatki tako prihajajo neposredno iz preverjenih virov, hkrati pa jih tudi sami navzkrižno primerjamo že od začetka delovanja (4. 3. 2020). Razlike v objavljenih podatkih se po navadi pojavijo zato, ker so bili zajeti ob različnih urah dneva.
+Most data for the previous day is collected at 11:59 pm (tests, confirmed cases ...), and hospitalization data is mostly obtained by 9 am every day for all hospitals. **Our data is usually updated between 10.00 and 12.00.**
+When we publish updated daily data, it is available on all our distribution channels (CSV, REST, website), and we also report it on social networks ([Facebook](https://www.facebook.com/COVID19Sledilnik) and [Twitter](https://twitter.com/sledilnik)).
 
-(Gl. tudi vprašanje So podatki verodostojni?)</p>
 </details>
 
 <details>
-  <summary>Zakaj poleg trenutno hospitaliziranih ne prikazujete tudi, koliko ljudi je bilo na določen dan sprejetih v hospitalizacijo?</summary>
+  <summary id=data-differences>Several portals display numbers of infected people that are different from yours. Why?</summary>
+
+Sledilnik uses only validated and official data reported daily by the NIPH and all Slovenian hospitals treating COVID-19. Our data thus comes directly from verified sources, and we have also cross-compared information from the very beginning (4 March 2020). Differences usually occur because different media and portals obtain the data at different times of the day or use dubious methodology. See also [Are your data and visualizations reliable?](/en/FAQ/#data-reliability) 
```

Comment:
Ali je smiselno prevajati kratico imena inštitucije NIJZ v NIPH? V about_en.md imamo kar originalno NIJZ.

Reply:
https://www.nijz.si/en/nijz uporablja kratico NIJZ, polno dolgo ime pa prevedeno.

Reply:
```suggestion
Sledilnik uses only validated and official data reported daily by the National Institute of Public Health (NIJZ) and all Slovenian hospitals treating COVID-19. Our data thus comes directly from verified sources, and we have also cross-compared information from the very beginning (4 March 2020). Differences usually occur because different media and portals obtain the data at different times of the day or use dubious methodology. See also [Are your data and visualizations reliable?](/en/FAQ/#data-reliability) 
```